src/components/generic/ThemeButton/ThemeButton.jsx

```diff
@@ -0,0 +1,48 @@
+import React, {JSX} from 'react';
+import PropTypes from 'prop-types';
+import webexComponentClasses from '../../helpers';
+
+/**
+ * ThemeButton component
+ *
+ * @param {object} props  Data passed to the component
+ * @param {Function} props.action  Adapter control callback
+ * @param {JSX.Element} props.children  List of children
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {boolean} props.isDisabled Flag indicating button enabled/disabled
+ * @param {string} props.title Tooltip to be displayed
+ * @param {string} props.type  Button type - default/join/cancel/ghost
+ *
+ * @returns {object}  JSX of the element
+ */
+export default function ThemeButton({
+  action,
+  children,
+  className,
+  isDisabled,
+  title,
+  type,
+}) {
+  const cssClasses = webexComponentClasses(`-${type}-button`, className);
```

Comment:
the base class should be `simply` button, you can add optional prefixed classes based on type, eg: `button--{type}`